app/js/controllers.js

```diff
@@ -41,4 +41,26 @@ angular.module('CompBrowser.controllers', [])
     ];
 
     $scope.orderProp = 'order';
+})
+
+//example controller for testing - THIS ISN" USED
```

Comment:
This is the controller that is being tested. It just checks the length of strings.